extensions/common/context.cc

```diff
@@ -551,5 +550,21 @@ bool sanitizeBytes(std::string* buf) {
   return modified;
 }
 
+// Used for `destination_service` fallback. Unlike elsewhere when that fallback
+// to workload name, this falls back to "unknown" when the canonical name label
+// is not found. This preserves the existing behavior for `destination_service`
+// labeling. Using a workload name as a service name could be potentially
+// problematic.
+std::string getServiceNameFallback() {
+  const auto local_node_buffer = extractLocalNodeFlatBuffer();
```

Comment:
this is not used in the code, is it done for a side-effect?

Reply:
it was sloppy typing without a compiler while participating in that meeting. i wanted to get general clearance before i found a way to a machine that could easily build the proxy.

Reply:
adopted @kyessenov 's suggestion.